root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -195,6 +196,27 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
 
             this.changeOtherRelationshipCredits = {source: ko.observable(false), target: ko.observable(false)};
             this.selectedRelationshipCredits = {source: ko.observable('all'), target: ko.observable('all')};
+
+            this.tooShortBeginYearError = debounce(function () {
+                const relationship = self.relationship();
+                if (!relationship.begin_date) {
```

Comment:
This *should* always be defined: https://github.com/metabrainz/musicbrainz-server/blob/dbfa033383754f3632aa53421b776ad1f5074adf/root/static/scripts/relationship-editor/common/fields.js#L65-L65

Do you remember what issue occurs or which test fails if the check is removed?

Reply:
No, but I removed it and it seems to work so I guess there wasn't one!